### DISCUSSION TITLE
Is doks multilingual?

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#55] [Ideas]
Congratulations on release of new theme! I wonder if adding content two or more languages is an option?

**Comment by h-enk** [comment]
Doks is multilingual. For, Hugo supports multiple languages out of the box: https://gohugo.io/content-management/multilingual/

**Reply by CicoZhang** [reply]
Would be nice if some demo code can show how to make doks multilingual, especially for people who are not so geek like me. 😄  
Really impressed by the simple style, thank you for making this template.

**Reply by epogrebnyak** [reply]
Here is an example of adding a language to Doks theme, feel free to explore. The source files are here: https://gitlab.com/epogrebnyak/doks-howto and the resulting site is https://epogrebnyak.gitlab.io/doks-howto/#translation

@h-enk, there some possible enhacement to the theme with respect for translation - but I think ther can be better discussed in issues. One is that there is not language switch in the theme yet, so even with translation the result is just a new language version and `/<lang>/` and another one is that one needs `title` in config.toml, not `params.toml` to make upper left "main" menu to work. There are other, lesser issues too, so far I took note of them in Russian at demo homepage.

![изображение](https://user-images.githubusercontent.com/9265326/106573420-a67b0f00-654a-11eb-8c2a-a294934dbe81.png)

**Reply by h-enk** [reply]
@epogrebnyak, nice work! I will have a closer look at it later.

**Reply by CicoZhang** [reply]
Thank you! I'll have a look!

**Comment by epogrebnyak** [comment]
Thank you, hope it is useful demo! I was not even sure I'd manage to
complete the demo, as there are little traps in Hugo you can fall at first
and spend a lot tiof figuring away out.

 I had to recall the translations in Academic theme and more recently in
LoveIt, CodeIT blogging themes that are bilingual from a start.

Can make a How-To or something similar for the docs, once a demo settles
and reviews.

вт, 2 февр. 2021 г., 12:01 Henk Verlinde <notifications@github.com>:

**Comment by RonenR** [comment]
Doks now supports translations, multilingual i18n out of the box:
https://getdoks.org/docs/how-to/extra-features/multilingual-mode/

**Reply by hawkerm** [reply]
This page no longer exists and the [new page is blank](https://getdoks.org/docs/guides/i18n/)?

I tried following the hugo instructions here: https://gohugo.io/content-management/multilingual/#translation-of-strings

I created a new `i18n` folder at the root of my project and added a `en.toml` file with:

```toml
[get_started]
other = "Start here"
```

But it didn't override the 'Example Guide' definition found in `doks-core\i18n\en.toml`... like I would have expected...

**Reply by h-enk** [reply]
Maybe the old docs could be of any help : [multilingual-mode](https://v0-4-2--getdoks.netlify.app/docs/reference-guides/multilingual-mode/)
